examples/flight-booker.zig

```diff
@@ -186,9 +186,8 @@ const App = struct {
             const leave_date = app.leave_date orelse break :enable_book;
             const return_date = app.return_date orelse break :enable_book;
 
-            if (leave_date.year <= return_date.year and
```

Comment:
original logic fails in a number of ways

eg, it thinks that 

12 dec 2023 is > 11 dec 2024
12 jan 2023 > 11 feb 2023
etc

I hate dates !!! 

Lucky for us. there is a "day of the year" field in the struct_tm